pkg/configauditreport/io.go

```diff
@@ -29,17 +29,17 @@ type Reader interface {
 
 	// FindReportByOwner returns a v1alpha1.ConfigAuditReport owned by the given
 	// kube.Object or nil if the report is not found.
```

Comment:
I think you missed updating the comment part here. It is still referring to kube.Object

Reply:
Good catch! Thanks. I've updated Go docs as well.